server/game/cards/14.5-AHD/FearlessSkirmisher.js

```diff
@@ -13,8 +13,8 @@ class FearlessSkirmisher extends DrawCard {
                 token: {
                     activePromptTitle: 'Choose a dishonored token',
                     mode: TargetModes.Token,
-                    tokenCondition: token => {
-                        return token.dishonored;
+                    cardCondition: card => {
+                        return card.dishonored;
```

Comment:
Why did you change this?

Reply:
because this was a hack anyway, just forget to change the other part. is fixed now